http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/app/WdsAppInstallSpec.scala

```diff
@@ -38,6 +38,7 @@ class WdsAppInstallSpec extends BaseAppInstallSpec {
       "wds.environmentBase=live," +
       "config.resourceGroup=mrg," +
       "config.applicationInsightsConnectionString=applicationInsightsConnectionString," +
+      "config.aks.vpaEnabled=false," +
```

Comment:
wouldn't this always make it be false? i.e. shouldn't we only make it false if there is no tags saying it would be true?

Reply:
ha wait, this is a test, ignore my comment :)